packages/commons/src/EmbeddedResourceAxeProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Deque.AxeCore.Commons
+{
+    public class EmbeddedResourceAxeProvider : IAxeScriptProvider
```

Comment:
I think it would probably be good for us to normalize the naming here while we're moving stuff around anyway:

* It would be nice for this to end in `AxeScriptProvider` for consistency
* The fact that we're using an embedded resource file is an implementation detail, not really a part of the contract of the class. I think maybe just `EmbeddedAxeScriptProvider` or `BundledAxeScriptProvider` would be a more succinct name for what a user of the class needs to know

Reply:
Updated with the new, agreed upon name `BundledAxeScriptProvider`